packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -995,19 +995,26 @@ export class AzureCommunicationCallAdapter<AgentType extends CallAgent | BetaTea
 
   /* @conditional-compile-remove(call-transfer) */
   private transferRequested(args: TransferRequestedEventArgs): void {
+    const acceptanceDeadline = new Date(new Date().getTime() + 5000).getTime();
```

Comment:
do we think that 5sec is enough time here? on another note, should this be in the code snippet for contoso as a best practice?

Reply:
No, the code snippet to use a popup would be removed and I would mention that request needs to be accepted or rejected right away